project/cmake/modules/FindVAAPI.cmake

```diff
@@ -29,6 +29,10 @@ elseif(VAAPI_INCLUDE_DIR AND EXISTS "${VAAPI_INCLUDE_DIR}/va/va_version.h")
   unset(vaapi_version_str)
 endif()
 
+if(VAAPI_VERSION_STRING VERSION_LESS 0.34)
+message(FATAL_ERROR "VAAPI minimum version 0.34 not found")
+endif()
+
 if(NOT VAAPI_FIND_VERSION)
   set(VAAPI_FIND_VERSION 0.38.0)
```

Comment:
the issue here is that we don't support AUTO mode for optional deps like we do in autotools.
An optional dep should only be (silently) disabled/ignored if it was not explicitly enabled by the user.

So if -DENABLE_VAAPI=ON was passed, we have to fail if the required version wasn't found.
I might have time to look into this tomorrow.